index.html

```diff
@@ -99,8 +100,15 @@
                             <tr>
                                 <th class="sortable text-nowrap" data-field="id" data-source="other">Unit# <i class="fa fa-arrows-v" aria-hidden="true"></i></th>
                                 <th class="sortable text-nowrap w-auto" data-field="name" data-source="other">Unit Name <i class="fa fa-arrows-v" aria-hidden="true"></i></th>
-                                <th class="w-auto">Lvl 5 Support</th>
-                                <th class="w-25 text-nowrap align-bottom"><div class="input-group flex-nowrap"> <div class="input-group-prepend"> <span class="input-group-text" id="addon-wrapping">Supported Units</span> </div> <input id="otherFilterSupported" class="form-control  w-100" type="text" label="otherFilteredLabel" placeholder="Filter"> </div> </th>
```

Comment:
here i just changed the class `w-25 text-nowrap align-bottom` to `w-auto align-bottom min-input-col-width`